src/afl-fuzz-run.c

```diff
@@ -709,6 +711,8 @@ u8 trim_case(afl_state_t *afl, struct queue_entry *q, u8 *in_buf) {
 
     });
 
+    if (orig_len != q->len) queue_testcase_retake(afl, q, orig_len);
```

Comment:
personal preference would be {} around if blocks :)

Reply:
done